### ISSUE TITLE
{   "message": "\"The requested name is valid, but no data of the requested type was found. (management.azure.com:443)\"" }

### ISSUE THREAD

**Issue by peteMorgan007** [microsoft/AzureStorageExplorer#7696]
{
  "message": "\"The requested name is valid, but no data of the requested type was found. (management.azure.com:443)\""
}

**Comment by peteMorgan007** [comment]
sign in error in Azure Storage explorer

**Comment by peteMorgan007** [comment]
can anyone help here

**Comment by MRayermannMSFT** [comment]
@peteMorgan007 thanks for reaching out.  Can you please follow these steps (https://learn.microsoft.com/en-us/troubleshoot/azure/azure-storage/storage-explorer-troubleshooting?tabs=Windows#authentication-logs) to gather **Authentication Logs**. Please make sure to follow all the steps, including enabling **Verbose Authentication Logging**, and ideally clearing the contents of the ServiceHub/logs folder before starting the Storage Explorer session where you reproduce the sign in failure.

Thanks!

**Comment by MRayermannMSFT** [comment]
Closing due to lack of response. This issue will no longer be monitored. If you need further help, we recommend opening a support ticket via the Azure portal. Alternatively you can open a new issue here.
